core/src/main/java/fr/sncf/osrd/standalone_sim/result/ElectrificationConditionsRange.java

```diff
@@ -16,7 +18,7 @@
  * and switches to "thermal" mode. Then used_mode = "thermal" and seen_mode = "25000". */
 public class ElectrificationConditionsRange {
     public final double start;
-    public final double stop;
+    public double stop;
```

Comment:
Why final or not final ?

Reply:
I made it not final so that I could update the stop if the next range computed in fact holds the same value

Reply:
Thanks for the explanation !